app/main/views/frameworks.py

```diff
@@ -1249,16 +1257,9 @@ def legal_authority(framework_slug):
         "name": "legal_authority",
         "hint": {"text": LegalAuthorityForm.HINT},
         "classes": "govuk-radios--inline",
-        "items": govuk_options(LegalAuthorityForm.OPTIONS)
+        "items": govuk_options(LegalAuthorityForm.OPTIONS),
+        "errorMessage": list(errors.values())[0]['errorMessage'] if errors else None
```

Comment:
Is this the only we way can extract the error message from the `errors` dict? I know we only have 1 field in this form but It'd be good if we could retrieve by field name...

Reply:
good point, fixed in 4a29640.